docs/try_argo_cd_locally.md

```diff
@@ -6,7 +6,7 @@
 
 Follow these steps to install `Kind` for local development and set it up with Argo CD.
 
-To run an Argo CD development environment [review the developer guide for running locally](../developer-guide/running-locally).
```

Comment:
The link works correctly on the docs page. 

https://argo-cd.readthedocs.io/en/stable/try_argo_cd_locally/

Reply:
https://github.com/user-attachments/assets/71d2f54a-3a78-4eec-971c-af464af9f754

I don't think this works as intended

Reply:
nvm, i missed the `.md`, afterwards it works as intended. the benefit is it will also work in the general markdown too.